deps/rabbitmq_aws/Makefile

```
@@ -1,16 +1,18 @@
 PROJECT = rabbitmq_aws
-PROJECT_DESCRIPTION = A minimalistic AWS API interface used by rabbitmq-autocluster (3.6.x) and other RabbitMQ plugins
-PROJECT_REGISTERED = rabbitmq_aws
+PROJECT_DESCRIPTION = AWS API library used by rabbitmq_peer_discovery_aws
+PROJECT_MOD = rabbitmq_aws
```

I think this should be
```suggestion
PROJECT_MOD = rabbitmq_aws_app
```

I get the following when trying to start rabbitmq
```
Error during startup: {error,
                       {rabbitmq_aws,
                        {bad_return,
                         {{rabbitmq_aws,start,[normal,[]]},
                          {'EXIT',
                           {undef,
                            [{rabbitmq_aws,start,[normal,[]],[]},
                             {application_master,start_it_old,4,
                              [{file,"application_master.erl"},
                               {line,295}]}]}}}}}}
```

---

no sorry, I was accidentally on a main from yesterday, 8ea901b1e553e363a57e4aea64334152967d9a90

it is indeed fixed by cd8029ef543944ddc19dd6069f7cb71a6dbcfdc7

---

I did exactly what @gomoripeti has suggested before marking the suggestion as resolved.